src/components/Avatar/__stories__/index.stories.tsx

```diff
@@ -14,9 +14,6 @@ const Template: Story<ComponentProps<typeof Avatar>> = args => (
 )
 
 export const Default = Template.bind({})
-Default.args = {
```

Comment:
Possible to have a proper exemple ?

Reply:
@chambo-e You're right story makes no sense, I rolled back to show avatar image into default story